experimental/sql/tools/rel_opt.py

```diff
@@ -3,17 +3,26 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+try:
+  from tools.IteratorsMlirConverter import IteratorsMlirConverter
+  mlir_loaded = True
+except ImportError:
+  mlir_loaded = False
+
 from xdsl.xdsl_opt_main import xDSLOptMain
```

Comment:
Notice this not obivous dependency: I have to import the `IteratorsMlirConverter` before import `xDSLOptMain`, since that would load the xDSL MlirConverter, registering the normal `mlir` module.

Reply:
Oh, this seems brittle. How about adding the following lines to a (to be created) [`__init__.py`](experimental/sql/src/ibis_mlir/__init__.py)?

```python
import mlir_iterators.ir
from xdsl import load_mlir_module
load_mlir_module(mlir_iterators)
```

This has to be done before anything that could potentially transitively call `ensure_mlir_module_loaded`, and having it in `__init__.py` seems like an easy and sure way to ensure that, no?

Reply:
That seems to have worked, the ordering does not matter anymore :+1: